src/platform/amd/renoir/lib/clk.c

```diff
@@ -0,0 +1,141 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+/*
+// Copyright(c) 2020 AMD.
+
+// Author: Basavaraj Hiregoudar<basavaraj.hiregoudar@amd.com>
+*/
+#include <sof/common.h>
+#include <sof/lib/clk.h>
+#include <sof/lib/cpu.h>
+#include <sof/lib/memory.h>
+#include <sof/lib/notifier.h>
+#include <sof/sof.h>
+#include <sof/spinlock.h>
+#include <platform/chip_registers.h>
+
+static struct freq_table platform_cpu_freq[] = {
+	{600000000, 600000 },
+};
+
+STATIC_ASSERT(NUM_CPU_FREQ == ARRAY_SIZE(platform_cpu_freq),
+				invalid_number_of_cpu_frequencies);
+
+static SHARED_DATA struct clock_info platform_clocks_info[NUM_CLOCKS];
+
+void  get_response_from_smu(void)
+{
+	uint32_t retValue = 0;
+	uint32_t count = 0;
+	do {
+		retValue = acp_reg_read_via_smn(MP1_SMN_C2PMSG_93, retValue, sizeof(int));
+		if (count > 0x007fffff) {
+			count = 0;
+		}
+		count++;
```

Comment:
Do we need a timeout here to avoid looping forever ? i.e. if the ret value does not change as expected.